openapi.yaml

```diff
@@ -171,6 +597,393 @@ components:
           schema:
             $ref: '#/components/schemas/ErrorResult'
   schemas:
+    BookId:
+      type: string
+      description: Book ID
+      example: '26'
+    BookType:
+      type: string
+      enum:
+        - book
```

Comment:
Forgot to mention this one. We usually do plural for the resource type.

Reply:
done

Reply:
Oh, I mean the value `book` -> `books`. Sorry for the confusion.

Reply:
done.